src/YamlValidator/InputProcessor.cs

```diff
@@ -33,33 +34,32 @@ public static RootCommand GetRootCommand()
             var inputFilePath = result.GetValueForOption(pathOption);
 
             // either file or folder must be passed
-            var pathType = string.Empty;
-            if (string.IsNullOrEmpty(inputFilePath))
+            if (string.IsNullOrWhiteSpace(inputFilePath))
             {
                 result.ErrorMessage = "The input is invalid, input must be a filepath to a yaml file \\" +
                 "or a folder path to a folder of yaml files";
             }
             else if (!Directory.Exists(inputFilePath) && !File.Exists(inputFilePath))
             {
-                result.ErrorMessage = "The input path does not exist";
+                result.ErrorMessage = $"The path '{inputFilePath}' does not exist";
             }
             else if (Directory.Exists(inputFilePath))
             {
                 if (Directory.GetFiles(inputFilePath, $"*{Constants.YamlFileExtension}").Length == 0)
                 {
-                    result.ErrorMessage = "The input folder does not contain any yaml files";
+                    result.ErrorMessage = $"The folder {inputFilePath} does not contain any yaml files";
```

Comment:
Should this have single quotes around the path like lines 44 and 57?

Reply:
Good point. Done